src/modules/__test__/add-remove.test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { listen, addToLi } from '../addtask.mjs';
+import remove from '../remove.mjs';
+
+// Add Test
+
+describe('addToLi Function', () => {
+  test('add Li element', () => {
+    document.body.innerHTML = '<ul id="todoList"></li>';
+
+    const todoList = document.querySelector('#todoList');
+
+    addToLi('name', 1, 'trashImg', 'taskList', todoList);
+    const list = document.querySelectorAll('#todoList li');
+    expect(list).toHaveLength(1);
+  });
+});
+
+describe('listen Function', () => {
+  test('Click add Task', () => {
+    document.body.innerHTML = '<form id="add" class="task"><input id="addTask" type="text" name="addTask" value="Feed dogs" placeholder="Add to your list..."><button type="submit" id="enter"></button></form><ul id="todoList"></li>';
```

Comment:
- [ ] I believe you want to create `<li>` at the end of line `24`, if so, kindly add the beginning `<li>` tag instead of `<ul>`.

Reply:
Thanks, Same as the previous error, it needs to be an `<ul>` to be exactly like the 'index.html' element its mocking.
We will make that changes.